EpsilonAuxBMS/AuxBMSTests.adoc

```diff
@@ -0,0 +1,83 @@
+= Aux BMS Tests
+
+== HeartBeat Test
+----
+Elapsed 1000ms{
+	Expect 1 Heartbeat CAN messages (0x720)
```

Comment:
Generally, when testing something that's supposed to happen regularly, you would want to test you receive more than one.